src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -19,7 +18,7 @@ public class Drivetrain extends SubsystemBase {
     RelativeEncoder rightEncoder = frontRighttDrive.getEncoder();
 
     DifferentialDrive drive = new DifferentialDrive(frontLeftDrive, frontRighttDrive);
-
+    Gyro gyro = new Gyro();
```

Comment:
I'd use a different variable name for the gyro object

Reply:
Done